python/cairo-ec/src/cairo_ec/curve/secp256k1.cairo

```diff
@@ -359,3 +371,21 @@ func try_recover_public_key{
     );
     return (public_key_point=res, success=1);
 }
+// @notice Converts a public key point to the corresponding Ethereum address.
+// @param x The x coordinate of the public key point.
+// @param y The y coordinate of the public key point.
+// @return The Ethereum address.
+func public_key_point_to_eth_address{
```

Comment:
```suggestion
func public_key_point_to_eth_address_be{
```

It's going to be _extremely_ confusing having two functions with the same name, one that returns an eth_address in a big-endian felt representation and one in a little-endian address.